mmv1/products/dlp/DiscoveryConfig.yaml

```diff
@@ -256,6 +256,7 @@ properties:
                           - 'SENSITIVITY_LOW'
                           - 'SENSITIVITY_MODERATE'
                           - 'SENSITIVITY_HIGH'
+                          - 'SENSITIVITY_UNKNOWN'
```

Comment:
Is the value `SENSITIVITY_UNKNOWN` a new value? I wonder what is the reason to add it.